sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -191,43 +189,21 @@ class MapboxMap internal constructor(
     onStyleLoaded: Style.OnStyleLoaded? = null
   ) {
     this.style = style
-    var resourceCount = styleExtension.resourceCount
-    styleExtension.sources.forEach {
-      if (it is GeoJsonSource) {
-        it.bindTo(style)
-        it.addOnGeoJsonParsedListener(object : OnGeoJsonParsed {
-          override fun onGeoJsonParsed(source: GeoJsonSource) {
-            resourceCount--
-            if (resourceCount == 0) {
-              onStyleLoaded?.onStyleLoaded(style)
-            }
-            it.removeOnGeoJsonParsedListener(this)
-          }
-        })
-      } else {
-        it.bindTo(style)
-        resourceCount--
-      }
-    }
     styleExtension.images.forEach {
```

Comment:
this is duplicate below at line L198

Reply:
could you please clarify what do you mean here?

Reply:
sorry, it showed the wrong suggestion somehow. Pointing to correct one.